repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/CaseTypeLiteACLEntity.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+
+@Table(name = "case_type_user_role")
+@Entity
+public class CaseTypeLiteACLEntity extends Authorisation implements Serializable {
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "case_type_id", nullable = false)
+    private CaseTypeLiteEntity caseType;
```

Comment:
Do we need the link to the case type entity at all? I feel it would be more straightforward if we only had top-down links for those entities, so case type has an ACL, but the ACL doesn't have a link back to the case type.

Reply:
removing reverse links to parent entities